sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2022-2023.bpmn

```diff
@@ -2125,7 +2125,7 @@ dmnFullTimeX (comes from dmnFullTime)</bpmn:text>
     <bpmn:textAnnotation id="TextAnnotation_0mz1bk7">
       <bpmn:text>IN
 studentDataRelationshipStatus
-studentDataHasDependents
+studentDataHasDependants
```

Comment:
As discussed please keep the existing variables as it is without being renamed. 

![image](https://github.com/user-attachments/assets/0eb9f853-d518-434e-bb98-cae8ba195762)